brave/brave-browser
# Enable vaapi support

Partially fixes #1024 

By default browser will blacklist all drivers on linux for vaapi decoding. 

To enable it , need to pass `--ignore-gpu-blacklist` in order to get it working since libva has problems atm with intel GPUs. I am hoping that it will be fixed in the future versions. There are a lot of activities in upstream in this.
Only adding this support on 64 bit build at the moment.


Based on this upstream commit: https://github.com/chromium/chromium/commit/31225b9c5f3f685d65f742dc129241c30c32157c

## Need to add libva 2.4 as a build and runtime dependency.

## Test Plan:
* Check if build succeeds.
* Run brave with `--ignore-gpu-blacklist` flag and see if the video is playing with hardware decoders or not. ( video needs to be supported by hardware)

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on



## Reviewer Checklist:
